app/tests/src/test_ingest_bem_pdfs.py

```diff
@@ -66,6 +66,10 @@ def mock_elements():
     ]
 
 
+def chunk_starting_with(chunks: list[Chunk], content: str):
+    return next(c for c in chunks if c.content.startswith(content))
```

Comment:
Using this function to avoid flaky test failure due ordering of returned chunks from the DB query, but it's curious b/c it only fails locally for s3 paths.